packages/sources/polygon-test/src/endpoint/tickers.ts

```diff
@@ -0,0 +1,115 @@
+import { EndpointTypes } from './router'
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { makeLogger } from '@chainlink/external-adapter-framework/util/logger'
+
+const logger = makeLogger('Polygon Tickers Logger')
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin'],
+    type: 'string',
+    description:
+      'The symbol of the currency to query. The full list of options can be found here [Physical Currency list](https://www.alphavantage.co/physical_currency_list/) or [Cryptocurrency list](https://www.alphavantage.co/digital_currency_list/)',
+    required: true,
+  },
+  quote: {
+    aliases: ['to', 'market'],
+    type: 'string',
+    description:
+      'The symbol of the currency to convert to. The full list of options can be found here [Physical Currency list](https://www.alphavantage.co/physical_currency_list/) or [Cryptocurrency list](https://www.alphavantage.co/digital_currency_list/)',
+    required: true,
+  },
+} as const
```

Comment:
After the typescript upgrade and the fw bump past 0.25, we want to move to using the satisfies operator here like 
```suggestion
} satisfies InputParameters
```